### ISSUE TITLE
User set images don't seem to load on Chrome on Linux (Debian 12) 

### ISSUE THREAD

**Issue by abclution** [web-launcher/web-launcher#32]
### Describe the bug

First off, lovely extension. Really excited to start using it!

### Steps to Reproduce

Tried setting an image (upload) for a shortcut, tried both .png and .jpg and neither seem to work.
The location of the image, chrome-extension://akomlegpokabommpdjfmhnbdcnaefmdo/skroutz-logo.png is created but when I check the plugin folders on disk it doesnt seem to be there. 

The only thing I have non standard is I do not have a chrome disk cache (symlinked to /dev/null) but I do have all the local website object stores enabled. 

### Version

3.3.0

### Additional context

I worked around it by linking it to a online image but just thought I'd drop by and give a whats up :)

**Comment by jwr1** [comment]
You're right. I just checked it, and it's not working for me either. I've been working on the next major version (https://github.com/sourcetab/sourcetab/tree/v4), but It's going pretty slow, so I don't know if I'll fix it directly or just wait for v4. Either way, as another alternative, you could use a file to data URL service ([like this one](https://www.adminbooster.com/tool/data_uri)) and paste the URL directly into the image URL field; this will work as an "upload". Anyway, thanks for reporting the issue (and also the review I'm assuming was yours).

**Comment by abclution** [comment]
Great suggestion and worked like a charm. :)
